docker/test/performance-comparison/Dockerfile

```diff
@@ -30,7 +30,7 @@ RUN apt-get update \
             tzdata \
             vim \
             wget \
-    && pip3 --no-cache-dir install clickhouse_driver scipy \
+    && pip3 --no-cache-dir install 'clickhouse-driver>=0.1.5' scipy \
```

Comment:
I hope that the docker image will be updated, however this is not mandatory, since it is likely already have 0.1.5